kratos/utilities/force_and_torque_utils.cpp

```diff
@@ -0,0 +1,75 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
```

Comment:
Format and missing authorship.

Reply:
@matekelemen feel free to add yourself as author!